fulp_modules/features/toys/code/plushies.dm

```diff
@@ -87,3 +87,25 @@
 	icon = 'fulp_modules/features/toys/icons/toys.dmi'
 	icon_state = "beefman"
 	squeak_override = list('sound/effects/meatslap.ogg'=1)
+
+//Do your cooldown changes here.
+#define BEEFPLUSHIE_COOLDOWN_TIME (2 MINUTES)
+
+/obj/item/toy/plush/beefplushie/living_beefplushie
```

Comment:
this typepath sucks, the beefplushie is already implied by it being a subtype of /obj/item/toy/plush/beefplushie